### ISSUE TITLE
i am using on rtx 2060 6gb ...although it was working b4 on automatic111 extension but now no more ? any solution to this 

### ISSUE THREAD

**Issue by thegreenscreenstation** [timothybrooks/instruct-pix2pix#59]
RuntimeError: CUDA out of memory. Tried to allocate 30.00 MiB (GPU 0; 6.00 GiB total capacity; 4.95 GiB already allocated; 0 bytes free; 5.02 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

**Comment by Dreamz-Dziner** [comment]
Yes. It was working fine few hours back but then strangely of a sudden it gives the "Cuda out of memory".
